khmer/_khmer.cc

```diff
@@ -308,9 +352,66 @@ static PyGetSetDef khmer_Read_accessors [ ] = {
 };
 
 
+static
+PyObject *
+khmer_Read_getattr(khmer_Read_Object *self, PyObject *attr_name)
```

Comment:
Is this necessary, since we have a `PyGetSetDef` defined?
https://github.com/dib-lab/khmer/pull/1492/files#diff-944b784821ddf1048fc73488ee2ee675R329